frontend/src/components/Map/Map.js

```diff
@@ -102,7 +107,7 @@ function Map({ config, hidePopup }) {
 
       function onMouseMove(e) {
         if (e?.features?.length > 0) {
-          if (hoveredFeatureId !== e.features[0].id) {
+          if (hoveredFeatureId !== e.features[0].id && !popupExpanded) {
```

Comment:
I think you could move this `&& !popupExpanded` up one line to the `if (e?.features?.length > 0) {` and remove the `if (!popupExpanded) {` check below, since none of this block wants to run if the popup's already expanded.